pgcli/main.py

```diff
@@ -1626,6 +1634,25 @@ def cli(
 
         sys.exit(0)
 
+    if ping_database:
+        results = None
+        try:
+            results = list(pgcli.pgexecute.run("SELECT 1"))
+        except Exception:
```

Comment:
I wonder what exception we could get, here. If pgcli cannot connect to the server (or use the database), we'll fail when calling `PGCLI.connect()` a few blocks before, not here.

Reply:
You're right, @dbaty, it's extremely unlikely, but not impossible. I've seen this once in my life, and not with postgresql; database server was up, but the transaction log was full. I'm ok with keeping the try/catch.

Reply:
> You're right, @dbaty, it's extremely unlikely, but not impossible. I've seen this once in my life, and not with postgresql; database server was up, but the transaction log was full. 

@j-bennet : then it would be useful to log the exact error we got, instead of the misleading "could not connect to the database".